travis.sh

```diff
@@ -64,18 +64,19 @@ if [ ! "$CATKIN_PARALLEL_TEST_JOBS" ]; then export CATKIN_PARALLEL_TEST_JOBS="$C
 if [ ! "$ROS_REPOSITORY_PATH" ]; then export ROS_REPOSITORY_PATH="http://packages.ros.org/ros-shadow-fixed/ubuntu"; fi
 if [ ! "$ROSDEP_ADDITIONAL_OPTIONS" ]; then export ROSDEP_ADDITIONAL_OPTIONS="-n -q -r --ignore-src"; fi
 if [ ! "$CATKIN_TOOLS_BUILD_OPTIONS" ]; then
-  if [ "ROS_DISTRO" = "hydro" ]; then
+  if [ "$ROS_DISTRO" = "hydro" ]; then
     export CATKIN_TOOLS_BUILD_OPTIONS="-iv --summarize --limit-status-rate 0.002"
   else
     export CATKIN_TOOLS_BUILD_OPTIONS="--summarize --limit-status-rate 0.002"
   fi
 fi
+if [ "$SUDO_PIP" = false ]; then export SUDO_PIP=""; else export SUDO_PIP="sudo -H"; fi
 echo "Testing branch $TRAVIS_BRANCH of $REPOSITORY_NAME"
 # Setup pip
 # FIXME: need to specify pip version to 6.0.7 to avoid unexpected error
 # https://github.com/jsk-ros-pkg/jsk_robot/pull/523#issuecomment-164699366
-sudo easy_install 'pip==6.0.7'
-sudo pip install -U -q pip setuptools
+$SUDO_PIP easy_install 'pip==6.0.7'
+$SUDO_PIP pip install -U -q pip setuptools
```

Comment:
@wkentaro I still don't think so. 

> But the SUDO_PIP variable cannot be reused, right? In current implementation, it is only for sudo prefix for pip install command.

Yes. This is what I mean. reusing `sudo` occurs unexpected effect. So it should not be overwritten in for example `.travis.yml`.
 
> It seems that --as-root option is overwritten if they are duplicated

This is the specific behavior of `rosdep`. We didn't understand this behavior  untill we executed `rosdep install` with duplicated arguments, right?